### PR TITLE
Add combiner recipes for Cyclic's nether/end ores

### PR DESCRIPTION
**Pull request by josephcsible**
The cost for redstone and lapis is lower than for the normal ores because Cyclic's versions of these have much lower yields, averaging only 2.2 redstone and 4.4 lapis per ore, even with fortune 3.

### CODE REVIEW

File: src/main/java/mekanism/common/integration/MekanismHooks.java [hunk 1]
```diff
@@ -173,6 +186,49 @@ public void loadOCDrivers()
 		} catch(Exception e) {}
 	}
 
+	@Method(modid = CYCLIC_MOD_ID)
+	public void registerCyclicCombinerOreRecipe(String ore, int quantity, Item input2, String outputName)
+	{
+		Item outputItem = ForgeRegistries.ITEMS.getValue(new ResourceLocation(CYCLIC_MOD_ID, outputName));
+		if(outputItem == Items.AIR) {
```
Comment (thiakil) [inline src/main/java/mekanism/common/integration/MekanismHooks.java]: that's not the default key as far as I can find (unless it has changed recently)

**Comment by thiakil** [comment]
> averaging only 2.2 redstone and 4.4 lapis per ore

Please make it the maximum possible yield or max +1

**Comment by josephcsible** [comment]
> Please make it the maximum possible yield or max +1

Why? As long as it's more expensive than the average yield, it can't be an infinite loop. Also, the costs of the vanilla combiner recipes aren't like that. For example, for vanilla lapis lazuli ore and a fortune 3 pickaxe, the average yield is 13.2, the maximum yield is 32, and the combiner cost is 16.

(I'll fix the two inline comments tonight.)

File: src/main/java/mekanism/common/integration/MekanismHooks.java (hunk 1 above)
Comment (josephcsible) [reply src/main/java/mekanism/common/integration/MekanismHooks.java]: Oops, you're right.

**Comment by thiakil** [comment]
> As long as it's more expensive than the average yield
needs to be what machine processing of the ore outputs

**Comment by josephcsible** [comment]
To my knowledge, no machines in either Mekanism or Cyclic let you do better processing of those redstone and lapis ores.

**Pull request merged**